### PR TITLE
Add draft template to docs

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add draft template to docs.

#### Changes

- Create draft template
- Reference template in DEVELOPMENT.md

#### Testing

Run locally (checked that draft key in Front Matter hides template, checked links in markdown)

##### Regressions

...

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/draft-template/_index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Requirements
+
+Use a requirements subheading to list requirements/prerequisites.
+
+1. Requirement 1
+2. Requirement 2
+
+## Sections
+
+When possible, divide long documents into separate files, each with its own folder and `_index.md`. Use the `weight`tag in the [Front Matter](https://gohugo.io/content-management/front-matter/) to manually sort sections if necessary. If not, they will be sorted alphabetically.
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:23]: nit; spacing between `weight` and tag

File: doc/content/draft-template/_index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Subheadings
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:25]: Maybe good to explicitly state that we use Title Case for headings?

File: doc/content/draft-template/_index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Since the title is a h1, everything in the content is at least h2 (##). Use subheadings to break up sections and provide a high level overview of steps the user should follow.
+
+## Notes
+
+Use the `>` markdown code, and include the word note to distinguish it from a code block.
+
+>Note: if you need help with any CLI command, use the `--help` flag to get a list of subcommands, flags and their description and aliases.
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:33]: Speaking of which, good to introduce the short codes; `cli-only` and `tts` are used quite a lot

File: doc/content/draft-template/_index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Images
+
+Screenshots should be inserted using the figure shortcode:
+
+{{< figure src="screenshot.png" alt="I am a screenshot" >}}
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:39]: Add instructions on how to take screenshots

I'm not sure what the instructions were exactly, but I thought it was using Chromium (Chrome, Brave, etc) developer tools, set display to "Laptop with HiDPI screen", then use the colons button for "Capture full size screenshot":

Screenshot 2020-06-03 at 08 43 35

File: doc/content/draft-template/_index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+To render images without styling, use standard markdown syntax:
+
+![I am a picture of Wienke](wienke.jpeg)
+
+## Shortcodes
+
+Explicitly call this product "The Things Stack". Not "the stack" etc. You can use the shortcode {{% tts %}} which will expand to "The Things Stack".
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:47]: Thou shalt use the shortcode

File: doc/content/draft-template/_index.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Syntax Highlighting
+
+See the following examples for types of syntax highlighting.
+
+### Console Button
```
Comment (johanstokking) [inline doc/content/draft-template/_index.md:53]: Empty line under all headers; this and below

**Comment by benolayinka** [comment]
@johanstokking pushed a fixup commit but I can't autosquash because the CHANGELOG changed between my first commit and last fixup commit (3.8.2 is released). I'm not sure how to rebase without it duplicating old CHANGELOG entries, let me know if you know!

**Pull request merged**